server/server.js

```diff
@@ -7,6 +8,14 @@ app.use(express.json());
 app.use('/api/trails', bikeTrailsRouter);
 app.use('/api/moreInfo', bikeTrailInfoRouter);
 
+// app.get('/', (req, res) => {
+
+// })
+
+app.get('/api', (req, res) => {
```

Comment:
This mock endpoint can be removed now, right?

Reply:
correct, this can be taken out